<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtzerkaluDialog</class>
 <widget class="QDialog" name="OtzerkaluDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mirroring parameters</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox_">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>331</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_">
    <item row="0" column="1">
     <widget class="QLineEdit" name="SaveDirLineEdit_"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Save to:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Rec Level:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="ChooseDirButton_">
      <property name="text">
       <string>...</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QCheckBox" name="FromOtherSite_">
      <property name="text">
       <string>Download contents from other sites</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QSpinBox" name="RecursionLevel_">
      <property name="specialValueText">
       <string>âˆž</string>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_</sender>
   <signal>accepted()</signal>
   <receiver>OtzerkaluDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_</sender>
   <signal>rejected()</signal>
   <receiver>OtzerkaluDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
